### PR TITLE
Enhancement/359 update mini mesos api

### PR DESCRIPTION
**Pull request by philwinder**

### CODE REVIEW

File: system-test/src/main/java/org/apache/mesos/elasticsearch/systemtest/TasksResponse.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public TasksResponse(String schedulerIpAddress, int nodesCount, String nodesStat
         this.schedulerIpAddress = schedulerIpAddress;
         this.nodesCount = nodesCount;
         this.nodesState = nodesState;
-        await().atMost(60, TimeUnit.SECONDS).until(new TasksCall());
+        await().atMost(60, TimeUnit.SECONDS).pollInterval(5L, TimeUnit.SECONDS).until(new TasksCall());
```
Comment (mwl) [inline system-test/src/main/java/org/apache/mesos/elasticsearch/systemtest/TasksResponse.java]: 5 seconds interval is a lot, isn't it?

File: system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/ExecutorSystemTest.java [hunk 2]
```diff
@@ -67,6 +63,10 @@ public static void beforeClass() {
         }
     }
 
+    private static List<Container> getExecutors() {
```
Comment (mwl) [inline system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/ExecutorSystemTest.java]: Sounds like something that should stay on TestBase?

File: system-test/build.gradle [hunk 3]
```diff
@@ -20,14 +20,16 @@ dependencies {
     compile 'com.github.docker-java:docker-java:1.3.0'
     compile 'com.mashape.unirest:unirest-java:1.4.5'
     compile 'com.jayway.awaitility:awaitility:1.6.3'
-    compile 'com.github.ContainerSolutions:mini-mesos:0.2.10'
+    compile 'com.github.ContainerSolutions:mini-mesos:bb53e37569'
+    compile 'com.google.code.findbugs:annotations:3.0.0'
```
Comment (mwl) [inline system-test/build.gradle]: I added this to `/build.gradle` on #370.

**Comment by philwinder** [comment]
Passed!
https://ci.infra.container-solutions.com/view/All/job/ElasticsearchPullRequests/162/console

**Pull request merged**